app/src/main/res/layout/fragment_quick_settings_dialog_sheet.xml

```diff
@@ -3,36 +3,35 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<androidx.constraintlayout.widget.ConstraintLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:id="@+id/quick_action_sheet"
-        android:orientation="vertical"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:contentDescription="@string/quick_settings_sheet"
-        android:background="?foundation">
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/quick_action_sheet"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?foundation"
+    android:contentDescription="@string/quick_settings_sheet"
+    android:orientation="vertical">
```

Comment:
ConstraintLayouts don't have orientation. It won't hurt though. It's just extraneous.